migrate_game.rb

```diff
@@ -182,6 +182,16 @@ def repair(game, original_actions, actions, broken_action)
         return
       end
     end
+    if game.active_step.is_a?(Engine::Game::G1856::Step::BuySellParShares) ||
```

Comment:
we're not gonna run this, and will just pin, so let's get rid of it

Reply:
That sounds good to me